OpenVisualCloud/Media-Transport-Library
# ptp: optimize phc2sys system clock offset less than 300ns.

To optimize phc2sys system clock offset, PTP delta must be optimized first.

Test network card: i225-LM

Test steps:
  1. Start master clock by command
  `ptp4l -i $INF_NAME -4 -l 6 -m --logSyncInterval -3 --tx_testamp_timeout 1000`

  2. Start IMTL sample app by command
  `sudo ./build/app/RxTxApp --ptp --phc2sys --pi --config_file $CONFIG_FILE --lcores 1,2,3`

Expect MTDEVSTATE output like:
```
  MT: PTP(0): system clock offset max 200
  MT: PTP(0): delta avg 3, min -11, max 16, cnt 80
```
The MTDEVSTATE output before optimization:
```
  MT: PTP(0): system clock offset max 3131
  MT: PTP(0): delta avg 1155, min 565, max 3735, cnt 80
```
